src/components/Identity.vue

```diff
@@ -257,7 +257,10 @@ export default {
     update() {
       const url = window.Urls['api:identity-detail'](this.identity.id);
       const data = this.identity;
-      return this.$http.put(url, data);
+      const req = this.$http.put(url, data);
+      // emit the Promise created by the $http call so that the state of the upload can be tracked if so desired
+      this.$emit('upload-started', req);
```

Comment:
An apparently small thing that I'm gonna be a bit of a hardass about: can you change all these cases of "upload" to "save"? "Upload" as a term implies file uploads to me very strongly, and as this is just POST/PUTting JSON to the server, it is _not that_. Without that name change, I will find this code hard to follow every time in the future that I revisit it.

Reply:
"Save" usually implies putting something directly on a local hard drive to me, and this is not that, but your point about file uploading is very fair. Kind of want to try out something silly like `transmission-started` then but I guess "save" should be fine

Reply:
"Transmission" is great! "Request" would work too?

I live in the web world enough that "save" usually means "to a remote" for me, so, "save" works for me, but I'm not _dedicated_ to it.